src/screens/LoginScreen.tsx

```diff
@@ -20,70 +26,110 @@ type Props = {
-        <View
-          style={[
-            containerStyles.fill,
-            containerStyles.centeredContent,
-            containerStyles.pageContainer,
-          ]}>
-          <Image source={bpPassportImage} />
-          <View
-            style={[
-              {
-                width: '80%',
-              },
-            ]}>
-            <PageHeader style={{textAlign: 'center'}}>
-              <FormattedMessage id="login.scan-your-passport" />
-            </PageHeader>
+        style={[containerStyles.fill, {backgroundColor: colors.grey4}]}>
+        <StatusBar barStyle="dark-content" />
+        <ScrollView showsVerticalScrollIndicator={false}>
+          <View style={[{marginTop: 40}]}>
+            <View style={[styles.loginContainer, {marginBottom: 10}]}>
+              <Image source={bpPassportImage} />
+              <PageHeader style={{textAlign: 'center', marginBottom: 22}}>
+                <FormattedMessage id="login.have-a-paper" />
+              </PageHeader>
+
+              <Button
+                style={[styles.primaryButton]}
+                buttonColor={colors.blue2}
+                title={intl.formatMessage({id: 'login.scan-passport'})}
+                onPress={() => {
+                  navigation.navigate(SCREENS.SCAN_BP_PASSPORT)
+                }}
+              />
+            </View>
+
+            <View style={[styles.loginContainer, {marginBottom: 18}]}>
+              <PageHeader style={{textAlign: 'center', marginBottom: 18}}>
+                <FormattedMessage id="login.no-bp-passport" />
+              </PageHeader>
+              <Button
+                style={{
+                  backgroundColor: colors.green1,
+                  width: '100%',
+                }}
+                title={intl.formatMessage({id: 'login.get-started'})}
+                onPress={() => {
+                  navigation.navigate(SCREENS.CONSENT)
+                }}
+              />
+            </View>
+
+            <View style={[{marginHorizontal: 24}]}>
+              <BodyText>
+                <FormattedMessage id="login.by-using-app" />{' '}
+                <BodyText style={{marginBottom: 24}}>
+                  <BodyText
+                    style={{color: colors.blue2}}
+                    onPress={() => {
+                      Linking.openURL('https://simple.org/patient-privacy')
+                    }}>
+                    <FormattedMessage id="login.privacy-policy-link" />{' '}
+                  </BodyText>
+                  <BodyText>
+                    <FormattedMessage id="general.and" />{' '}
+                  </BodyText>
+                  <BodyText
+                    style={{color: colors.blue2}}
+                    onPress={() => {
+                      Linking.openURL('https://simple.org/digitalprinciples/')
+                    }}>
+                    <FormattedMessage id="login.terms-of-use-link" />
+                  </BodyText>
+                  <BodyText>.</BodyText>
```

Comment:
I don't think this needs to be inside another BodyText tag

Reply:
This is from James current work I believe.